src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java

```diff
@@ -231,13 +234,22 @@ public final void setNativeTextBlockLayout(android.text.Layout layout, int leftP
 		_topTextBlockPadding = topPadding;
 	}
 
+	@Override
+	protected boolean drawChild(Canvas canvas, View child, long drawingTime) {
```

Comment:
Is this useful ?

Reply:
No, see reply above to Mathieu :D

Reply:
Well you removed it while `I was reviewing :)